tests/traits/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -41,14 +42,19 @@ trait A {
   }
 }
 
-trait B extends otherTrait {
+trait B
+  extends otherTrait
```

Comment:
Shouldn't the line break come after `extends`?

Reply:
@czosel seems no, good catch :+1: